apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -581,6 +581,23 @@ type EventCollectionFeatureConfig struct {
 	// CollectKubernetesEvents enables Kubernetes event collection.
 	// Default: true
 	CollectKubernetesEvents *bool `json:"collectKubernetesEvents,omitempty"`
+
+	// UnbundleEvents enables collection of Kubernetes events as individual events.
+	// Default: false
+	UnbundleEvents *bool `json:"unbundleEvents,omitempty"`
```

Comment:
should these parameters have `+optional` ?

Reply:
It's implicit already as nullable type and `omitempty` but added as others are explicitly flagged as well.